pkgs/by-name/hp/hplip/package.nix

```diff
@@ -362,13 +356,7 @@ python3Packages.buildPythonApplication {
           bsd2
           gpl2Plus
         ];
-    platforms = [
-      "i686-linux"
-      "x86_64-linux"
-      "armv6l-linux"
-      "armv7l-linux"
-      "aarch64-linux"
-    ];
+    platforms = lib.attrNames hplipPlatforms;
```

Comment:
Previously the assertion was predicated on `withPlugin`. Does that imply the supported platforms are different when that parameter is enabled?

should this be something like:

```nix
platforms =
  if withPlugin then
    builtins.attrNames hplipPlatforms
  else
    lib.platforms.linux;
```